### PR TITLE
CUJ Tweaks for I can terminate my relationship with Google

### LINKED ISSUE #1101: [CWS CUJs] I can terminate my relationship with Google
[Friction Log](https://docs.google.com/document/d/1u5s51NGGPUhLCkAqaL8rueWpeSFSvrfnlOU2chlimZs/edit?usp=sharing)

### PR DESCRIPTION
**Pull request by AmySteam**
Fixes #1101 

**Changes proposed in this pull request:**

- Add links to unpublishing instructions
- Add screenshot to unpublish button

CC: @CharyCh

:star: **Staged Preview links**

[Deleting CWS account](https://deploy-preview-1145--developer-chrome-com.netlify.app/docs/webstore/account-deletion/)
[How to unpublish items](https://deploy-preview-1145--developer-chrome-com.netlify.app/docs/webstore/faq/#faq-listing-03)

### CODE REVIEW

File: site/en/docs/webstore/faq/index.md [hunk 1]
```diff
@@ -193,11 +193,13 @@ quickly.
 
 ### How can I remove my listing from the store? {: #faq-listing-03 }
 
-Visit [https://chrome.google.com/webstore/developer/dashboard][115]. Click the item you want to
+Visit the [developer dashboard][115]. Click the item you want to
```
Comment (awfuchs) [inline site/en/docs/webstore/faq/index.md:196]: 115? The links in this file are out of control. Can you please file a bug to reconcile the links in this file later? Thanks!

**Review by awfuchs — APPROVED** [review]
LGTM but please note my request to open a bug. Thanks!

File: site/en/docs/webstore/faq/index.md (hunk 1 above)
Comment (samthor) [reply site/en/docs/webstore/faq/index.md:196]: As a side note @AmySteam, we don't like the numbered [115] links—this is just an artifact of the way this content was imported. So we definitely want to clean them up to be named or inline (I actually don't have a preference).

Comment (AmySteam) [reply site/en/docs/webstore/faq/index.md:196]: Yeah that makes sense. I'll start converting the links I find from now on. Thanks for the tip 🙏🙂🌻

**Pull request merged**